Marlin/ultralcd.cpp

```diff
@@ -2799,16 +2804,17 @@ void lcd_update() {
 
         #if ENABLED(DOGLCD)  // Changes due to different driver architecture of the DOGM display
           static int8_t dot_color = 0;
-          dot_color = 1 - dot_color;
-          u8g.firstPage();
-          do {
+          if (!drawing_screen) {
+            u8g.firstPage();
+            dot_color = 1 - dot_color;
+          }
             lcd_setFont(FONT_MENU);
             u8g.setPrintPos(125, 0);
             u8g.setColorIndex(dot_color); // Set color for the alive dot
             u8g.drawPixel(127, 63); // draw alive dot
             u8g.setColorIndex(1); // black on white
```

Comment:
Or an option to use it as an indicator for how full the buffer is? half way on the display mean buffer is half full.. sort of a progress indicator except for a complet bar. (unless it takes the same cpu time to draw a line ?)